lib/alipay/service.rb

```diff
@@ -209,17 +209,14 @@ def self.create_forex_trade_wap_url(params, options = {})
       request_uri(params, options).to_s
     end
 
-    # Alipay Commerce
-    # alipay doc: https://global.alipay.com/service/merchant_QR_Code/15
-
     CREATE_MERCHANT_QR_CODE_REQUIRED_PARAMS = %w( biz_type biz_data )
     CREATE_MERCHANT_QR_CODE_REQUIRED_BIZ_DATA_PARAMS = %w( secondary_merchant_industry secondary_merchant_id secondary_merchant_name trans_currency currency )
     def self.create_merchant_qr_code(params, options = {})
       params = Utils.stringify_keys(params)
       check_required_params(params, CREATE_MERCHANT_QR_CODE_REQUIRED_PARAMS)
       biz_data = nil
 
-      if params['biz_data'].present?
+      if params['biz_data']
```

Comment:
`.present?` is only work in rails, so to make this gem working fine even with pure ruby, just check using the params will do